docs/RELEASE-NOTES.md

```diff
@@ -5,6 +5,13 @@ the extension's page on mediawiki.org:
 
 * https://www.mediawiki.org/wiki/Extension:Semantic_Tasks
 
+## Version 2.0.1
+
+Released on 2020-11-26
```

Comment:
Oh cmon you could have waited 4 days for it to be a 1 year release ;)

Reply:
Thought about this, too. Realized this too late. :|